benchexec/tools/sv_benchmarks_util.py

```diff
@@ -0,0 +1,28 @@
+# This file is part of BenchExec, a framework for reliable benchmarking:
+# https://github.com/sosy-lab/benchexec
+#
+# SPDX-FileCopyrightText: 2007-2020 Dirk Beyer <https://www.sosy-lab.org>
+#
+# SPDX-License-Identifier: Apache-2.0
+
+"""
+This module contains some useful functions related to tasks in the sv-benchmarks
```

Comment:
Please add a note about lack of guarantees of stability and no use for out-of-tree modules (as we discussed).

Reply:
fixed in 6f11a4cb9c66b01effc2b0a591ce95e368f99631